docs/standards/universal-profile/lsp0-erc725account.md

```diff
@@ -251,3 +253,32 @@ The ability to add new functions to the LSP0 is crucial for its extendibility po
 However, the account can declare support for new interface IDs after it has been extended. This is especially beneficial for contracts that check if a contract supports a specific interface ID before interacting with it.
 
 ![LSP0 Extended interfaceIds](/img/standards/lsp0/LSP0-Extended-Interfaces.jpeg)
+
+
+### LSP20 - Call Verification
+
+:::tip
+
+See the **[LSP20 - CallVerification](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-20-CallVerification.md)** standard for more information.
+
+Check the **[LSP0 functions](../smart-contracts/lsp0-erc725-account.md)** to see how verification of allowed calls to the account happens.
+
+:::
+
+The **LSP0ERC725Account** is an ownable contract that can be owned by different types of addresses, including EOAs, contracts like multisigs, KeyManagers, etc.. These owner contracts may have various functions and behaviors, which can create some challenges in terms of figuring out how the interaction with the account works.
+
+Previously, When the account is owned by a KeyManager, for example, only addresses allowed by the KeyManager should interact with it. They cannot directly interact with the account functions because only the KeyManager can call them as the owner. A similar issue arises when the account is owned by a voting contract, as voters must interact with the voting contract rather than the account directly.
```

Comment:
```suggestion
Previously, When the account was owned by a KeyManager, only addresses allowed by the KeyManager should interact with the account via the Key Manager. They cannot directly interact with the account functions because only the KeyManager can call them as the owner. A similar issue arises when the account is owned by a voting contract, as voters must interact with the voting contract rather than the account directly.
```